src/coldfront_plugin_api/management/commands/fix_allocation_users.py

```diff
@@ -0,0 +1,48 @@
+from django.core.management.base import BaseCommand
+from coldfront.core.allocation.models import AllocationUser
+from coldfront.core.project.models import Project, ProjectUser, ProjectUserStatusChoice, ProjectUserRoleChoice
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+class Command(BaseCommand):
+    help = "A command that adds users who have been assigned allocations to those allocations' project. Made to fix a \
+    database inconsistency where a user is assigned to an allocation, but may not have been assigned to the allocation's project. \
+    Run this command with no options to see the list of users with this inconsistency. Run with '--apply' to fix them."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--apply",
+            action="store_true", 
+            help="Applies changes to database to fix all allocation user inconsistencies"
+        )
+        return 0
```

Comment:
`return 0` is not necessary. Functions that have no `return` statement in Python `return None` automatically.